build/postgres-operator/install-extensions.sh

```diff
@@ -16,6 +16,16 @@ if [[ -n $STORAGE_ENDPOINT ]]; then
 	args+=(-endpoint "$STORAGE_ENDPOINT")
 fi
 
+# Checking the STORAGE_DISABLE_SSL env for backwards compatibility before 2.8.0
+if [[ -n $STORAGE_DISABLE_SSL ]]; then
+	args+=(-disable-ssl "$STORAGE_ENDPOINT")
```

Comment:
Do we need to have "$STORAGE_ENDPOINT  for -disable-ssl?

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1214/commits/25feab6fbb5daa5772708e2c673e28b9b16ce795